Cargo.lock

```diff
@@ -4107,6 +4129,16 @@ dependencies = [
  "zerocopy",
 ]
 
+[[package]]
+name = "libz-ng-sys"
```

Comment:
we don't already have libz?

i'm not sure the fix is worth the added dependencies

Reply:
We do, apparently this uses `libz-ng-sys` instead of `libz-sys/zlib-ng` (which is what we have in our dep tree).

Removed the feature that adds it now.